lblod/ember-rdfa-editor-lblod-plugins
# GN-4544: Always get decision title from public decisions

### Overview

Always get decision title from public decisions.
There was an issue where `?decisionTitle` was not always fetched by public decisions query correctly. This fixes that 🙏 

##### connected issues and PRs:

* https://binnenland.atlassian.net/browse/GN-4544

### Setup

1. Checkout
2. `npm run start`

### How to test/reproduce

1. Go to Besluit Sample
2. Try to insert reference
3. Change filter to the one depicted on the image
4. Observe that results are rendered correctly

![image](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/assets/769698/c9780dc8-895c-403b-868b-ed3b26b140b6)


### Challenges/uncertainties

Instead of using `?decisionTitle` in both `OPTIONAL` fragment I've used different names based on where `decisionTitle` is coming from and then `BOUND ( COALESCE`, works a charm


### Checks PR readiness
